backfill/2025-01-22-firefox-installer schema-error/README.md

```diff
@@ -0,0 +1,186 @@
+# 2025-01-22 Firefox installer schema errors
+
+Bug for error: https://bugzilla.mozilla.org/show_bug.cgi?id=1934302
+
+Bug for backfill: https://bugzilla.mozilla.org/show_bug.cgi?id=1941442
+
+In Firefox 134, some new fields were added to the Firefox installer telemetry.  This includes `firefox_installer.install` and `stub_installer` pings.  These pings go through different pipelines but both end up in `firefox_installer_live.install_v1`.
```

Comment:
```suggestion
In Firefox 134, some new fields were added to the Firefox installer telemetry. This includes `firefox_installer.install` and `stub_installer` pings. These pings go through different pipelines but both end up in `firefox_installer_live.install_v1`.
```

More out of curiosity: it looks like there are always two spaces after a period

Reply:
That's just how I learned to type sentences so I do it out of habit but markdown and a lot of things turn it into one space